packages/react-components/react-link/stories/Link/LinkAsSpan.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { Link } from '@fluentui/react-components';
+
+export const AsSpan = () => <Link as="span">Render as a span</Link>;
+
+AsSpan.parameters = {
+  docs: {
+    description: {
+      story: ['A Link can be rendered as an html `<span>`, in which case it will have `role="button"` set.'].join('\n'),
```

Comment:
I think you can remove the join and array

```suggestion
      story: 'A Link can be rendered as an html `<span>`, in which case it will have `role="button"` set.',
```

Reply:
IMO it would nice to add details on _why_ it should be used? (context from #29600)